tests/test_model.py

```diff
@@ -0,0 +1,157 @@
+# © MNELAB developers
+#
+# License: BSD (3-clause)
+
+import math
+from contextlib import contextmanager
+
+import numpy as np
+import pytest
+from edfio import Edf, EdfSignal
+
+from mnelab import Model
+
+
+@pytest.fixture
+def model():
+    """
+    Fixture to create a Model instance and attach a DummyView
+    """
+
+    # Dummy class for view
+    class DummyView:
+        @contextmanager
+        def _wait_cursor(self):
+            yield
+
+        def data_changed(self):
+            pass
+
+    model_instance = Model(view=DummyView())
+    return model_instance
```

Comment:
Here I would prefer to simply adapt the `data_changed` decorator so that it can handle cases where `view is None`.